identity/client/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "identity-frontend",
   "private": true,
-  "version": "0.0.0",
+  "version": "8.7.0-SNAPSHOT",
```

Comment:
Similar question to above, do we have to manually manage this version?

Reply:
Yes, it's not great. It's how the other webapps work though (e.g. [tasklist](https://github.com/camunda/camunda/blob/fc38c934dbb43f2b67bb10b2fdfada3dcdee2315/tasklist/client/package.json#L3), [operate](https://github.com/camunda/camunda/blob/fc38c934dbb43f2b67bb10b2fdfada3dcdee2315/operate/client/package.json#L3)).

Reply:
Ah I see, so in the other components they include an execution in their POM to set this as part of the pipeline ([REF](https://github.com/camunda/camunda/blob/fc38c934dbb43f2b67bb10b2fdfada3dcdee2315/tasklist/client/pom.xml#L41-L50)), can we also add this so we benefit from the same structure/flow?

Reply:
Oh, this is better!